articles/how-to-optimize-wordpress-website-speed/index.md

```diff
@@ -0,0 +1,224 @@
+the expense of performance. Pay special attention to theme optimization. Your
+WordPress theme should be elegant, lightweight, SEO friendly, fully
+responsive, and mobile optimized.
+
+### Optimize WordPress database
+
+Over time, the WordPress database will accumulate unnecessary data,
+clutter. This will slow down your website. The clutter could be post
+revisions, data that was created by already uninstalled plugins, and
+WordPress sessions. To optimize the database, you can use plugins. It is
+advisable to back up your database before trying to clean. Some of the
+plugins available for database optimization are
+[WP-Optimize](https://wordpress.org/plugins/wp-optimize/) and
+[WP-Sweep](https://wordpress.org/plugins/wp-sweep/).
+
+![](wp-optimize-database-optimization.jpeg)
+
+See above screenshot of WP-Optimize 3.1.2 free version database
+optimization
+
+### Use lazy loading for images and videos
+
+Lazy loading is a web optimization technique whereby, visible content is
+loaded first while contents appearing down the fold download and
+rendering is delayed. Instead of loading all images and videos, Lazy
+loading downloads and render only what is visible to the user's screen.
+As the user scrolls down the page, more content is loaded. This
+significantly increases the loading speed of the content that is visible
+to the user.
+
+### Limit the number of post revisions
+
+WordPress has a very good feature called post revisions. Whenever you
+update a page or a post, another version of the post is created and
+saved in the database. Overtime, post revisions can increase to a point
+whereby they slow your WordPress database. In the screenshot below, post
+revisions have accumulated to 93.
+
+![WordPress post revisions screenshot](wordpress-post-revisions-screenshot.jpeg)
+
+The solution for post revisions is to clean them from the database. You
+can use the WP-Optimize plugin to clean post revisions as shown in the
+screenshot below.
+
+![WP-Optimize post revisions cleanup](wp-optimize-post-revisions-cleanup.jpeg)
+
+See above screenshot of WP-Optimize 3.1.2 free version deleting posts
+revisions
+
+The next step after cleaning post revisions is to limit the number of
+post revisions saved for each post. This can be done by adding this line
+of code on the WordPress wp-config.php file.
+
+```php
+define('WP_POST_REVISIONS', 4 )
+```
+
+![WordPress wp-config editing](wp-config.php-edit.jpg)
+
+### Update PHP to the latest version
+
+PHP is opensource and undergoing active development to improve on
+performance, fix bugs, enhance security, and other features. Therefore,
+there is no doubt that the latest PHP version is more enhanced in terms
+of performance compared to the previous versions. Keep your PHP version
+updated to get the benefits. PHP update can be as simple as selecting
+the latest version for cPanel users. Check with your hosting provider on
+how to update the PHP version.
+
+### Use Content Delivery Network (CDN)
```

Comment:
You could expand here on the current state of PHP versions in WordPress. 

What's the minimum supported version and the max? Which versions are in development and which are common hosting providers using? When does support end for different PHP versions?